### PR TITLE
Packit: disable osh diff scan

### PR DESCRIPTION
**Pull request by lsm5**
Often takes too long and results are being ignored for now.

Ref: https://packit.dev/docs/configuration#osh_diff_scan_after_copr_build

### CODE REVIEW

**Comment by lsm5** [comment]
@rhatdan @ericcurtin @smooge hopefully that should do it. PTAL

**Review by ericcurtin — APPROVED** [review]

**Comment by ericcurtin** [comment]
I really need to find out how to make that "Merge on green build" button appear here, we have it in OSTree repo and it's super useful, the button says "Enable auto-merge" and will merge once builds pass.

**Comment by rhatdan** [comment]
I think we should still require one LGTM to pass as well.  Eventually would be nice to get to the point of requiring two.

**Pull request merged**
